recipes/civetweb/all/conanfile.py

```diff
@@ -77,18 +101,32 @@ def _configure_cmake(self):
         if self._cmake:
             return self._cmake
         self._cmake = CMake(self)
-        self._cmake.definitions["CIVETWEB_ENABLE_SSL"] = self.options.with_ssl
+
         if self.options.with_ssl:
             openssl_version = tools.Version(self.deps_cpp_info["openssl"].version[:-1])
```

Comment:
This would break with openssl greater then 3.x 🤔 maybe a validate check?

Reply:
It was already there before my PR, so I didn't touch it. I am not against taking the opportunity to do that as well. What would you recommend?

Reply:
It's the first PR since... let's not block this good work, maybe follow up with another PR :pray:

Reply:
@prince-chrismc Sure, will do!